src/Uno.AzureDevOps/Uno.AzureDevOps.Views/Content/ProjectItemDetails.xaml

```diff
@@ -458,16 +458,16 @@
 				</Grid>
 
 				<!-- Double click back button tutorial -->
-				<Grid Visibility="{Binding ShowDoubleBackTip, Converter={StaticResource TrueToVisible}, FallbackValue=Collapsed}"
-					  Background="{StaticResource Color02Brush}"
+				<Grid Background="{StaticResource Color02Brush}"
+					  Visibility="{Binding ShowDoubleBackTip, Converter={StaticResource TrueToVisible}, FallbackValue=Collapsed}"
```

Comment:
For large blocks of code that depends on particular visibility, it would make more sense to keep it as a top priority before the others in my opinion. (Easier to read when the entire block is collapsed also)